src/platform/site-wide/announcements/config/index.js

```diff
@@ -29,7 +29,7 @@ const config = {
       component: WelcomeToNewVAModal,
     },
     {
-      name: 'veterans-day-proclamation',
+      name: 'veterans-day-proclamation-edited',
```

Comment:
This is the value used to persist the banner's dismissal into localStorage. Since we're changing it, it'll show for everyone.